src/library/Uno.Material/Styles/Controls/v2/HyperlinkButton.xaml

```diff
@@ -49,15 +52,15 @@
             <StaticResource x:Key="HyperlinkButtonFontSize" ResourceKey="LabelLargeFontSize" />
             <StaticResource x:Key="HyperlinkButtonFontFamily" ResourceKey="MaterialRegularFontFamily" />
 
-            <StaticResource x:Key="HyperlinkButtonBackgroundOpacity" ResourceKey="NormalOpacity" />
+            <StaticResource x:Key="HyperlinkButtonBackgroundOpacity" ResourceKey="HyperlinkButtonNormalOpacity" />
             <StaticResource x:Key="HyperlinkButtonBackgroundOpacityPointerOver" ResourceKey="MediumOpacity" />
             <StaticResource x:Key="HyperlinkButtonBackgroundOpacityPressed" ResourceKey="LowOpacity" />
-            <StaticResource x:Key="HyperlinkButtonBackgroundOpacityDisabled" ResourceKey="NormalOpacity" />
-        </ResourceDictionary>
-    </ResourceDictionary.ThemeDictionaries>
+            <StaticResource x:Key="HyperlinkButtonBackgroundOpacityDisabled" ResourceKey="HyperlinkButtonNormalOpacity" />
 
-    <Thickness x:Key="HyperlinkButtonPadding">0</Thickness>
-    <x:Double x:Key="NormalOpacity">1</x:Double>
+			<Thickness x:Key="HyperlinkButtonPadding">0</Thickness>
+			<x:Double x:Key="HyperlinkButtonNormalOpacity">1</x:Double>
```

Comment:
we could just rename this to HyperlinkButtonBackgroundOpacity, and move it up
same for disabled

Reply:
like this? 

```      
<ResourceDictionary> 
        ...
        ...

        <x:Double x:Key="HyperlinkButtonBackgroundOpacity">1</x:Double>
        <StaticResource x:Key="HyperlinkButtonBackgroundOpacityPointerOver" ResourceKey="MediumOpacity" />
        <StaticResource x:Key="HyperlinkButtonBackgroundOpacityPressed" ResourceKey="LowOpacity" />
	<x:Double x:Key="HyperlinkButtonBackgroundOpacityDisabled">1</x:Double>

	<Thickness x:Key="HyperlinkButtonPadding">0</Thickness>
</ResourceDictionary>

Reply:
@Xiaoy312 is this what you meant? ^